api/documents/migrations/0001_initial.py

```diff
@@ -23,7 +22,6 @@ class Migration(migrations.Migration):
                     ),
                 ),
                 ("text", models.TextField()),
-                ("embedding", pgvector.django.VectorField(dimensions=10)),
             ],
```

Comment:
**issue:** create separate migration, DO NOT regenerate existing ones if it's not necessary (it's not this time)
1. update models
2. ./migrate.py makemigrations